### PR TITLE
fix: remove tailwind anti-pattern from the repo

### LINKED ISSUE #1522: refactor: remove anti-pattern from the repo
We spent a lot of time rewiring the hot repo after leaning into hardcoding pixels in tailwind. We may want to avoid this anti-pattern 

https://github.com/open-sauced/hot/pull/337

_Originally posted by @bdougie in https://github.com/open-sauced/insights/pull/1500#discussion_r1287280902_

### PR DESCRIPTION
**Pull request by RitaDee**
## Description
This pull request focuses on eliminating a Tailwind anti-pattern that was present in the repository's codebase. The changes made in this PR address the usage of the problematic Tailwind `w-[72px]`, replacing it with a more suitable and maintainable alternative. 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #1522
## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by OgDev-01 — APPROVED** [review]
LGTM

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-npx lint-staged
+#!/bin/sh
```
Comment (NsdHSO) [inline .husky/pre-commit:1]: Please could you revert these changes, I don't see anything change

File: components/atoms/CartIllustration/cart-illustration.tsx [hunk 2]
```diff
@@ -3,13 +3,9 @@ interface CartIllustrationProps {
...
-    <div
-      className={`inline-flex max-w-[176px] gap-1 items-center px-1.5 w-44 h-12 rounded-md border ${
-        classNames ?? classNames
-      }`}
-    >
-      <div className="h-[34px] w-[34px] border rounded bg-light-slate-5"></div>
-      <div className="w-[80px]  gap-2  rounded">
+    <div className={`inline-flex gap-1 items-center px-1.5 w-44 h-12 rounded-md border ${classNames ?? classNames}`}>
+      <div className="h-9 w-9 border rounded bg-light-slate-5"></div>
+      <div className="w-20  gap-2  rounded">
```
Comment (NsdHSO) [inline components/atoms/CartIllustration/cart-illustration.tsx:8]:
```suggestion
      <div className="w-20 gap-2 rounded">
```

Have you tried to run the format script ??

File: components/atoms/CartIllustration/cart-illustration.tsx [hunk 3]
```diff
@@ -3,13 +3,9 @@ interface CartIllustrationProps {
...
 const CartIllustration = ({ classNames }: CartIllustrationProps) => {
   return (
-    <div
-      className={`inline-flex max-w-[176px] gap-1 items-center px-1.5 w-44 h-12 rounded-md border ${
-        classNames ?? classNames
-      }`}
-    >
-      <div className="h-[34px] w-[34px] border rounded bg-light-slate-5"></div>
-      <div className="w-[80px]  gap-2  rounded">
+    <div className={`inline-flex gap-1 items-center px-1.5 w-44 h-12 rounded-md border ${classNames ?? classNames}`}>
```
Comment (NsdHSO) [inline components/atoms/CartIllustration/cart-illustration.tsx:6]: In the previous version, we have `max-w...` in the current version we don't have it. this is an intended feature ???

File: .husky/pre-commit (hunk 1 above)
Comment (RitaDee) [reply .husky/pre-commit:1]: > Please could you revert these changes, I don't see anything change

I encountered difficulties committing my changes, which led me to investigate the cause in the .husky/pre-commit file. To address the issue, I temporarily duplicated the line `npx lint-staged` and confirmed whether my commit would succeed. Upon successful validation, I proceeded with the commit and subsequently removed the duplicated line. This adjustment resolved the problem on my part.

File: components/atoms/CartIllustration/cart-illustration.tsx (hunk 2 above)
Comment (RitaDee) [reply components/atoms/CartIllustration/cart-illustration.tsx:8]: @NsdHSO I did try, but it isn’t fixing it. I will do the fix manually.

File: components/atoms/CartIllustration/cart-illustration.tsx (hunk 3 above)
Comment (RitaDee) [reply components/atoms/CartIllustration/cart-illustration.tsx:6]: I removed max-w-[176px] since it duplicates the width set by w-44, causing redundancy in the styling. What do you suggest?

**Review by brandonroberts — APPROVED** [review]
